lib/sigh/developer_center.rb

```diff
@@ -59,49 +59,53 @@ def maintain_app_certificate(force = nil)
         page_size = 500
 
         until has_all_profiles do
-          certs = post_ajax(@list_certs_url, "{pageNumber: #{page_index}, pageSize: #{page_size}, sort: 'name%3dasc'}")
+          bundle_id = Sigh.config[:app_identifier]
 
-          profile_name = Sigh.config[:provisioning_name]
+          certs = post_ajax(@list_certs_url, "{pageNumber: #{page_index}, pageSize: #{page_size}, sort: 'name%3dasc', search: 'name%3D#{bundle_id}%26type%3D#{bundle_id}%26status%3D#{bundle_id}%26appId%3D#{bundle_id}'}")
```

Comment:
Yeah, this is just Apples way of taking what's in the search form field and passing it along. I just kept it default, didn't test if passing less params would work.
![apple](https://cloud.githubusercontent.com/assets/1731082/7205841/c6857536-e4f1-11e4-8d2c-176cf38da97e.png)